# main.go does not have 25 lines. The panic reference will not work because it is from the annotated file.
fail godebugtester run -srclines=false main.go
fail contains stderr "main.go:7"
contains stderr "# godebug.exec"

fail godebugtester run -srclines=true -nodebugmsg main.go
contains stderr "main.go:7"
fail contains stderr "# godebug.exec"

-- go.mod --
module mod1
-- main.go --
package main
func main() {
	println(1)
	println(1)
	println(1)
	println(1)
	panic("panicking")
}
